src/main/java/org/isf/permissions/model/GroupPermission.java

```diff
@@ -56,9 +47,17 @@ public class GroupPermission extends Auditable<String> {
 
 	@NotNull
 	@ManyToOne
-	@JoinColumn(name = "GP_P_ID_A", referencedColumnName="P_ID_A")
+	@JoinColumn(name = "GP_P_ID_A", referencedColumnName = "P_ID_A")
 	private Permission permission;
 
+	public GroupPermission() {
```

Comment:
what is the purpose of this method without parameters?

Reply:
I've added a constructor with parametters. There where many places in the code where the default constructor without params was using. After adding the constructor, they where complaining, so I've added a no params constructor. I could refactor those places to use the Args constructor, but ...

To you want me to make a refactoring and replace usage like:

`
Permission permission...;
UserGroup userGroup...;

GroupPermission groupPermission = new GroupPermission();
groupPermission.setPermission(permission);
groupPermission.setUserGroup(userGroup);
`
with:
`
Permission permission...;
UserGroup userGroup...;

var groupPermission = new GroupPermission(userGroup, permission);
`

Or should I remove the GroupPermission cunstroctor with params I've added ?

Reply:
You can leave it as it is.